crates/vm/levm/src/memory.rs

```diff
@@ -51,6 +51,12 @@ impl Memory {
             .splice(offset..offset + len, value.iter().copied());
     }
 
+    pub fn store_n_bytes(&mut self, offset: usize, value: &[u8], size: usize) {
+        self.resize(offset + size);
+        self.data
+            .splice(offset..offset + size, value.iter().copied());
+    }
+
```

Comment:
What about renaming this function to `store_bytes` and removing the current one?

Reply:
I know that it is quite redundant but the opcodes that use the actual `store_bytes` don't want to send the size along the offset. They get the size from the value itself.
In the case of the opcode that uses `store_n_bytes` (see `generic_call()`) it only specifies the size because it doesn't want to store the whole value on memory, only up to a defined limit.

So the other opcodes should change from:
`current_call_frame.memory.store_bytes(dest_offset, &code);`
to
`current_call_frame.memory.store_bytes(dest_offset, &code, code.len());`

Anyway, it isn't a big deal. If you consider it is worth making the change despite this then I'll do it :)

Reply:
Yes! That's what I was thinking of.